doc/README.md

```diff
@@ -1,48 +1,45 @@
-BlueChi's documentation pages
-===========================
+# BlueChi's documentation pages
```

Comment:
Here the trademark isn't needed since its about `BlueChi`, not the project Eclipse BlueChi, right?

Reply:
I've added the trade mark to both files, thanks